src/Api.Rest.Tests/Dtos/Data/SearchConditionParserTest.cs

```diff
@@ -0,0 +1,202 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2021                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Api.Rest.Tests.Dtos.Data
+{
+	#region usings
+
+	using System;
+	using System.Collections.Generic;
+	using FluentAssertions;
+	using NUnit.Framework;
+	using Zeiss.PiWeb.Api.Rest.Dtos.Data;
+
+	#endregion
+
+	[TestFixture]
+	public class SearchConditionParserTest
+	{
+		#region members
+
+		private static IEnumerable<( string condition, bool result )> _TestCases = new[]
+		{
+			("123=[]", true),
+			("123=]", false),
+			("123=[", false),
+			("123=condition", false),
+			("123=[condition", false),
+			("123=condition]", false),
+			("123=[condit]ion", false),
+			("123=[condit]ion]", true),
+			("123=[condition[]]", true),
+			("123=[]condition]", true),
+			("123=[con[]dition]", true),
+			("123=[con[]dition]+124=[con[]dition]", true),
+			("123=[con[]dition]+124=[con[]dition", false)
+		};
```

Comment:
Since this test cases are only used by a single test method, can you annotate the test cases directly at the test method? This improves readability.

Reply:
Done